src/main/java/seedu/address/model/session/Session.java

```diff
@@ -43,6 +43,19 @@ public Fee getFee() {
         return fee;
     }
 
+    /**
+     * Checks if the {@code Session} slot overlaps with another session.
+     * @param otherSession the other session that is compared to.
+     */
+    public boolean isOverlapping(Session otherSession) {
```

Comment:
Coming up, will have to edit this to serve Recurring Sessions as well.
TODO for me

Reply:
Gotcha